docs/installation.rst

```diff
@@ -5,19 +5,21 @@ Installation instructions
 
 Non-python dependencies
 -----------------------
-Scaper has two non-python dependencies:
+Scaper has one non-python dependency:
+- FFmpeg: https://ffmpeg.org/
 
+If you are installing Scaper on Windows, you will also need:
 - SoX: http://sox.sourceforge.net/
-- FFmpeg: https://ffmpeg.org/
 
-On macOS these can be installed using `homebrew <https://brew.sh/>`_:
+If you are installing on Linux/macOS, the SoX dependency is taken care of via 
+[SoxBindings](https://github.com/pseeth/soxbindings).
```

Comment:
"On Linux/macOS SoX is replaced by SoxBindings which is significantly faster, giving better overall performance in Scaper. On these platforms SoxBindings is installed automatically when calling pip install scaper (see below)."